specification/storagecache/resource-manager/Microsoft.StorageCache/stable/2020-03-01/storagecache.json

```diff
@@ -0,0 +1,1776 @@
+        },
+        "location": {
+          "description": "Region name string.",
+          "type": "string"
+        },
+        "name": {
+          "description": "Name of Cache.",
+          "readOnly": true,
+          "$ref": "#/definitions/ResourceName"
+        },
+        "type": {
+          "description": "Type of the Cache; Microsoft.StorageCache/Cache",
+          "readOnly": true,
+          "type": "string"
+        },
+        "identity": {
+          "$ref": "#/definitions/CacheIdentity",
+          "description": "The identity of the cache, if configured."
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "description": "Properties of the Cache.",
+          "properties": {
+            "cacheSizeGB": {
+              "description": "The size of this Cache, in GB.",
+              "type": "integer"
+            },
+            "health": {
+              "description": "Health of the Cache.",
+              "readOnly": true,
+              "$ref": "#/definitions/CacheHealth"
+            },
+            "mountAddresses": {
+              "description": "Array of IP addresses that can be used by clients mounting this Cache.",
+              "readOnly": true,
+              "items": {
+                "type": "string"
+              },
+              "type": "array"
+            },
+            "provisioningState": {
+              "description": "ARM provisioning state, see https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/Addendum.md#provisioningstate-property",
+              "enum": [
+                "Succeeded",
+                "Failed",
+                "Cancelled",
+                "Creating",
+                "Deleting",
+                "Updating"
+              ],
+              "x-ms-enum": {
+                "name": "ProvisioningStateType",
+                "modelAsString": true
+              },
+              "type": "string"
+            },
+            "subnet": {
+              "description": "Subnet used for the Cache.",
+              "$ref": "#/definitions/URLString"
+            },
+            "upgradeStatus": {
+              "description": "Upgrade status of the Cache.",
+              "$ref": "#/definitions/CacheUpgradeStatus"
+            },
+            "networkSettings": {
+              "$ref": "#/definitions/CacheNetworkSettings",
+              "description": "Specifies network settings of the cache."
+            },
+            "encryptionSettings": {
+              "$ref": "#/definitions/CacheEncryptionSettings",
+              "description": "Specifies encryption settings of the cache."
+            },
+            "securitySettings": {
+              "$ref": "#/definitions/CacheSecuritySettings",
+              "description": "Specifies security settings of the cache."
+            }
+          },
+          "type": "object"
+        },
+        "sku": {
+          "description": "SKU for the Cache.",
+          "properties": {
```

Comment:
just confirm, sku name is string instead of enum?

Reply:
Yes, sku name is string.